synapse/lib/view.py

```diff
@@ -361,3 +372,14 @@ def pack(self):
         if self.parent is not None:
             trigs.extend(await self.parent.listTriggers())
         return trigs
+
+    async def trash(self):
+        '''
+        Delete the underlying storage for the view.
+
+        Note: this does not delete any layer storage.
+        '''
+        await s_hive.AuthGater.trash(self)
+
+        for (iden, _) in self.triggers.list():
+            self.triggers.delete(iden)
```

Comment:
Did we test to make sure that inherited triggers are not trashed when trashing a forked view?

Reply:
No, but it is pretty easy to tell from inspection.